sparkly/catalog.py

```diff
@@ -203,6 +205,71 @@ def set_table_property(self, table_name, property_name, value):
             table_name, property_name, value
         ))
 
+    def get_database_property(self, db_name, property_name, to_type=None):
+        """Read value for database property.
+
+        Args:
+            db_name (str): A database name.
+            property_name (str): A property name to read value for.
+            to_type (function): Cast value to the given type. E.g. `int` or `float`.
+
+        Returns:
+            Any
+        """
+        if not to_type:
+            to_type = str
+
+        value = self.get_database_properties(db_name).get(property_name)
+        if value is not None:
+            return to_type(value)
+
+    def get_database_properties(self, db_name):
+        """Get database properties from the metastore.
+
+        Args:
+            db_name (str): A database name.
+
+        Returns:
+            dict[str,str]: Key/value for properties.
+        """
+        properties = (
+            self._spark.sql('DESCRIBE DATABASE EXTENDED {}'.format(db_name))
+            .where(F.col('database_description_item') == 'Properties')
+            .select('database_description_value')
+            .first()
+        )
+
+        parsed_properties = {}
+
+        if properties:
+            for name, value in read_db_properties_format(properties.database_description_value):
+                parsed_properties[name] = value
+
+        return parsed_properties
+
+    def set_database_property(self, db_name, property_name, value):
+        """Set value for database property.
+
+        Args:
+            db_name (str): A database name.
+            property_name (str): A property name to set value for.
+            value (Any): Will be automatically casted to string.
+        """
+        property_name_blacklist = {',', '(', ')'}
```

Comment:
should this be a class level constants to be patchable if needed

Reply:
not sure, it doesn't make sense to change them without changing `read_db_properties_format`